CommonClasses/interface/DiPhotonInfo.h

```diff
@@ -25,7 +25,7 @@ namespace ExoDiPhotons
     bool isEEEE;
```

Comment:
The declaration `double rapidity;` should be moved after this line so that the definition of the `diphotonInfo_t` struct matches the branch definition. The branch names are just labels and ROOT does not perform any check that the labels are consistent with the C++ variables to which they refer.

Reply:
Hi Chris, I moved the variable declarations last.